README.md

```diff
@@ -30,6 +30,23 @@ The gradebook builds on the awesome work of @jwo and @matthiasak. Each student y
 
 Data is cached in localStorage so as not to DDoS TIYO (too much). Clicking the "Refresh Grades" button will scrape the site once again and store the new data.
 
+### Homework
+
+The Homework module builds upon the inspiration from @jah2488.  It allows communication to an authorized ruby process running on your computer to rapidly clone and setup git based projects on your computer.  This allows for easy cloning of PR's, Git Repos, Gists, and various other git related links.  For full documentation please read more about the underlying tool newline_hw [here](https://github.com/TIYDC/newline_hw#newlinehw).
+
```

Comment:
When the client is detected
![screenshot 2017-01-10 11 42 53](https://cloud.githubusercontent.com/assets/243989/21815276/0632cd9c-d72a-11e6-8847-993f3176b6a1.png)


When the client is missing
![screenshot 2017-01-10 11 42 35](https://cloud.githubusercontent.com/assets/243989/21815290/0fc70f94-d72a-11e6-9e1a-e16e1cd86754.png)